index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apple.com Clone [Project Backgrounds and Gradients]</title>
+    <link rel="stylesheet" href="style/style.css">
+    <link rel="stylesheet" href="style/Normalize.css">
+</head>
+
+<body>
+    <main>
+        <section class="top-section">
+                <h1>.</h1>
+            <nav class="nav-bar">
+                <button class="nav-bar-image"></button>
+                <button>Store</button>
+                <button>Mac</button>
+                <button>iPod</button>
+                <button>iPhone</button>
+                <button>iTunes</button>
+                <button>Support</button>
+                <div class="nav-input-wrap">
+                    <input type="search">
+                </div>
+            </nav>
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See how people are using</h2>
+            <h2>iPad in their unique ways to add to the world's story.</h2>
+            <h2 class="play">Watch the "Your Verse" film and explore the stories</h2>
+        </section>
+        <section class="image-tile">
+            <h1>.</h1>
+            <div></div>
+            <div></div>
+            <div></div>
```

Comment:
You can make your targeting more specific with classes or ids.

Reply:
Oh sorry i guess its a bug i will fix that now. Thanks